src/python/pants/engine/fs.py

```diff
@@ -295,11 +295,14 @@ def write_digest(
         Get(Digest, MergeDigests)` beforehand.
```

Comment:
This function is used for commands like `pants package ::` that write to `dist/`, files that aren't in the graph. From my testing, this seems fine? Logs just say `external invalidation: cleared 0 and dirtied 0 nodes for: {"y.pex"}`.

Reply:
Yep, noops are fine.